client/state/action-types.js

```diff
@@ -442,6 +442,14 @@ export const READER_LIST_UPDATE_DESCRIPTION = 'READER_LIST_UPDATE_DESCRIPTION';
 export const READER_LIST_UPDATE_FAILURE = 'READER_LIST_UPDATE_FAILURE';
 export const READER_LIST_UPDATE_SUCCESS = 'READER_LIST_UPDATE_SUCCESS';
 export const READER_LIST_UPDATE_TITLE = 'READER_LIST_UPDATE_TITLE';
+export const READER_FETCH_TAG_REQUEST = 'READER_FETCH_TAG_REQUEST';
+export const READER_FETCH_TAG_RECEIVE = 'READER_FETCH_TAG_REQUEST_RECEIVE';
+export const READER_FETCH_TAGS_REQUEST = 'READER_FETCH_TAGS_REQUEST';
```

Comment:
hmm.  I don't want to have such different naming for two apis that return the same shape (one is tag other is tags). Looking more at it, they are even the same URL (one is /read/tags, other is /read/tags/{tagSlug}).  

I'm gonna reduce it down to one action type with a ternary to decide the path!